AnkiDroid/src/main/java/com/ichi2/compat/CompatV19.java

```diff
@@ -39,6 +39,7 @@ public void setFullScreen(final AbstractFlashcardViewer a) {
         // Show / hide the Action bar together with the status bar
         SharedPreferences prefs = AnkiDroidApp.getSharedPrefs(a);
         final int fullscreenMode = Integer.parseInt(prefs.getString("fullscreenMode", "0"));
+        final boolean topBar = prefs.getBoolean("showTopbar", true);
```

Comment:
Isn't the default for the preference false?
```suggestion
        final boolean showTopBar = prefs.getBoolean("showTopbar", false);
```

Reply:
Default value is true, also in ```preferences.xml``` and in ```AbstractFlashcardViewer.java```